Bottom: a00140763e6d56523c1c0ccc8e8fb5d34671c7f2
Top:    4af14606f2e897f5c74c9b9f08d9c6474ecc3170
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-17 19:58:43 +0000

polygon.h: Reduce circ_segs to 8 for debug clarity


---

diff --git a/src/polygon.h b/src/polygon.h
index 369efea..0643765 100644
--- a/src/polygon.h
+++ b/src/polygon.h
@@ -34,7 +34,7 @@
 
 /* Implementation constants */
 
-#define POLY_CIRC_SEGS 40
+#define POLY_CIRC_SEGS 8 //40
 #define POLY_CIRC_SEGS_F ((float)POLY_CIRC_SEGS)
 
 #if 0
